src/Umbraco.Web/PropertyEditors/ValueConverters/MarkdownEditorValueConverter.cs

```diff
@@ -1,53 +0,0 @@
-﻿using System;
-using System.Web;
-using HeyRed.MarkdownSharp;
-using Umbraco.Core;
-using Umbraco.Core.Models.PublishedContent;
-using Umbraco.Core.PropertyEditors;
-using Umbraco.Web.Composing;
-using Umbraco.Web.Templates;
-
-namespace Umbraco.Web.PropertyEditors.ValueConverters
```

Comment:
Was this file supposed to be deleted?

Reply:
Yes, at some point this was duplicated, and also lives in Infrastructure